samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/facade/QuestionPoolFacadeQueries.java

```diff
@@ -317,32 +317,32 @@ public List getAllItemFacadesOrderByItemText(final Long questionPoolId,
 						     final String orderBy, final String ascending) {
 	    
 	  	// Fixed for bug 3559
-	    log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: orderBy=" + orderBy);  
+	    log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: orderBy = {}", orderBy);
 	    List list = getAllItems(questionPoolId);
 
-	    log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: size = " + list.size());
+	    log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: size = {}", list.size());
 	    HashMap hp = new HashMap();
 	    Vector origValueV;
 	    ItemData itemData;
 	    ItemFacade itemFacade;
 	    Vector facadeVector = new Vector();
 	    String text;
 	    for (int i = 0; i < list.size(); i++) {
-	    	log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: i = " + i);
+	    	log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: i = {}", i);
 	    	itemData = (ItemData) list.get(i);
 	    	itemFacade = new ItemFacade(itemData);
 	    	facadeVector.add(itemFacade);
-	    	log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: getItemId = " + itemData.getItemId());
-	    	log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: getText = " + itemData.getText());
+	    	log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: getItemId = {}", itemData.getItemId());
+	    	log.debug("QuestionPoolFacadeQueries: getAllItemFacadesOrderByItemText:: getText = {}", itemData.getText());
 	    	
 	    	// SAM-2499
-	    	text = formattedText.stripHtmlFromText( itemFacade.getText(), false, true ).trim();
+	    	text = formattedText.stripHtmlFromText(itemFacade.getText(), false, true);
```

Comment:
This looks like the only real necessary change in this file, it seems like it would be better to have all of these other logging changes on a separate jira, possibly with more changes?